### ISSUE TITLE
example for neural_network without activation.

### ISSUE THREAD

**Issue by lihuiknight** [aymericdamien/TensorFlow-Examples#175]
hi, in your example for neural-network.py, the function neural_net, layer_1 = tf.layers.dense(x, n_hidden_1) is without activation, in my opinion, this example is equal with a multinomial logistic regression, right?

**Comment by nova77** [comment]
Indeed. It should be `layers.dense(x, n_hidden_1, activation=tf.nn.relu)` (for instance).
